gtm/src/template-parameters.json

```diff
@@ -0,0 +1,404 @@
+            "paramValue": "convertedObjectIDs"
+          }
+        ],
+        "displayName": "Sends a conversion event to capture the clicked items.",
+        "name": "labelConvertedObjectIDs",
+        "type": "LABEL"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "eventType",
+            "type": "EQUALS",
+            "paramValue": "convertedFilters"
+          }
+        ],
+        "displayName": "Sends a conversation event to capture the filters a user uses when converting.",
+        "name": "labelConvertedFilters",
+        "type": "LABEL"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "eventType",
+            "type": "EQUALS",
+            "paramValue": "viewedObjectIDs"
+          }
+        ],
+        "displayName": "Sends a view event to capture the items.",
+        "name": "labelViewedObjectIDs",
+        "type": "LABEL"
+      },
+      {
+        "enablingConditions": [
+          {
+            "paramName": "eventType",
+            "type": "EQUALS",
+            "paramValue": "viewedFilters"
+          }
+        ],
+        "displayName": "Sends a view event to capture the filters.",
+        "name": "labelViewedFilters",
+        "type": "LABEL"
+      }
+    ]
+  },
+  {
+    "enablingConditions": [
+      {
+        "paramName": "eventType",
+        "type": "EQUALS",
+        "paramValue": "init"
+      }
+    ],
+    "displayName": "",
+    "name": "Init Setting",
+    "groupStyle": "NO_ZIPPY",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "displayName": "App ID",
+        "help": "The identifier of your Algolia application.",
+        "simpleValueType": true,
+        "name": "appId",
+        "type": "TEXT"
+      },
+      {
+        "displayName": "API Key",
+        "help": "The search API key of your Algolia application.",
+        "simpleValueType": true,
+        "name": "apiKey",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "enablingConditions": [
+      {
+        "paramName": "eventType",
+        "type": "EQUALS",
+        "paramValue": "init"
+      }
+    ],
+    "displayName": "Advanced Algolia Settings",
+    "name": "Advanced Algolia Settings",
+    "groupStyle": "ZIPPY_CLOSED",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "displayName": "Opt Out User",
+        "name": "userHasOptedOut",
+        "help": "Whether to exclude users from analytics.",
+        "macrosInSelect": true,
+        "simpleValueType": true,
+        "type": "SELECT",
+        "selectItems": [
+          {
+            "displayValue": "False",
+            "value": false
+          },
+          {
+            "displayValue": "True",
+            "value": true
+          }
+        ]
+      },
+      {
+        "displayName": "Region",
+        "name": "region",
+        "help": "The DNS server to target (support values: de, us).",
+        "simpleValueType": true,
+        "type": "TEXT"
```

Comment:
shouldn't this be a SELECT instead (only 2 values are available) ?

Reply:
Good call!